third_party/terraform/utils/iam_batching.go

```diff
@@ -0,0 +1,60 @@
+package google
+
+import (
+	"fmt"
+	"google.golang.org/api/cloudresourcemanager/v1"
+	"time"
+)
+
+const (
+	batchKeyTmplModifyIamPolicy = "%s modifyIamPolicy"
+
+	IamBatchingEnabled  = true
+	IamBatchingDisabled = false
+)
+
+func batchRequestModifyIamPolicy(updater ResourceIamUpdater, modify iamPolicyModifyFunc, config *Config, reqDesc string) error {
```

Comment:
```suggestion
func BatchRequestModifyIamPolicy(updater ResourceIamUpdater, modify iamPolicyModifyFunc, config *Config, reqDesc string) error {
```

Even though everything we do is in the same package, both other functions here are effectively private. We can show that this is the only fn we expect consumers to use by capitalising it as if it needed to be exported.